hudi/README.md

```diff
@@ -0,0 +1,71 @@
+# Agent Check: Hudi
+
+## Overview
+
+This check monitors [Hudi][1].
+It is compatible with Hudi [versions][2] `0.10.0` and above.
+
+## Setup
+
+### Installation
+
+The Hudi check is included in the [Datadog Agent][3] package.
+No additional installation is needed on your server.
+
+### Configuration
+
+1. [Configure][4] the [JMX Metrics Reporter][5] in Hudi:
+
+    ```
+    hoodie.metrics.on=true
+    hoodie.metrics.reporter.type=JMX
+    hoodie.metrics.jmx.host=<JMX_HOST>
+    hoodie.metrics.jmx.port=<JMX_PORT>
+    ```
+
+
+2. Edit the `hudi.d/conf.yaml` file, in the `conf.d/` folder at the root of your
+   Agent's configuration directory to start collecting your hudi performance data.
+   See the [sample hudi.d/conf.yaml][6] for all available configuration options.
+
+   This check has a limit of 350 metrics per instance. The number of returned metrics is indicated in the info page.
```

Comment:
```suggestion
   This check has a limit of 350 metrics per instance. The number of returned metrics is indicated in the info page.
```
info page? Is that the Datadog Agent Status page or Hudi?

Reply:
I think the datadog agent status page- this was generated from the new JMX integration template- should we update it?

Reply:
Might be worth clarifying, especially since if you're not using GUI agent, it's not really a page 😅

Reply:
Sure, I also updated the template here: https://github.com/DataDog/integrations-core/pull/10193/files